application.py

```diff
@@ -359,7 +368,7 @@ def contest(contest_id):
 
     # Ensure contest started or user is admin
     start = datetime.strptime(contest_info[0]["start"], "%Y-%m-%d %H:%M:%S")
-    if datetime.now() < start and not session["admin"]:
+    if datetime.utcnow() < start and not session["admin"]:
         return redirect("/")
 
     title = contest_info[0]["name"]
```

Comment:
Final review: lines 374-404 need to be changed, this is still the old format, currently causing an Internal Server Error